routers/products-router.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+var bodyParser = require('body-parser');
+const ProductsRouter = express.Router();
+const cors = require('cors')
+ProductsRouter.use(cors())
```

Comment:
Like this you are using this in every single route.

Reply:
done.